Libplanet.Headless/Hosting/LibplanetNodeService.cs

```diff
@@ -195,27 +196,31 @@ public class LibplanetNodeService<T> : BackgroundService, IDisposable
                 DifferentAppProtocolVersionEncountered = Properties.DifferentAppProtocolVersionEncountered,
             };
             var hostOptions = new Net.HostOptions(Properties.Host, shuffledIceServers, Properties.Port ?? default);
+            var swarmOptions = new SwarmOptions
+            {
+                BranchpointThreshold = 50,
+                StaticPeers = Properties.StaticPeers,
+                MinimumBroadcastTarget = Properties.MinimumBroadcastTarget,
+                BucketSize = Properties.BucketSize,
+                MaximumPollPeers = Properties.MaximumPollPeers,
+                TimeoutOptions = new TimeoutOptions
+                {
+                    MaxTimeout = TimeSpan.FromSeconds(50),
+                    GetBlockHashesTimeout = TimeSpan.FromSeconds(50),
+                    GetBlocksBaseTimeout = TimeSpan.FromSeconds(5),
+                },
+            };
 
+            var transport = NetMQTransport.Create(
+                Properties.SwarmPrivateKey,
+                appProtocolVersionOptions,
+                hostOptions,
+                swarmOptions.MessageTimestampBuffer).ConfigureAwait(false).GetAwaiter().GetResult();
```

Comment:
We may need to care of this case in future PR 🤔

Reply:
See also: https://stackoverflow.com/a/46027058

Reply:
I agree with the sentiment. Not sure about what the best solution would be. I'm leaning towards sync-over-async factory tho. 🙄